src/Sulu/Component/Content/Types/Block/HiddenBlockSkipper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Types\Block;
+
+use Sulu\Component\Content\Compat\Block\BlockPropertyType;
+
+class HiddenBlockSkipper implements BlockSkipperInterface
```

Comment:
would it be possible to extract this functionality and include it into a `2.1.*` release? that would really simplify supporting this in the `SuluHeadlessBundle` 🙂